docs/components/movement-sensor/merged.md

```diff
@@ -0,0 +1,104 @@
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "<your-sensor-name>",
+      "type": "movement_sensor",
+      "model": "merged",
+      "attributes": {
+        "position": ["<your-gps-sensor-name-1>", "<your-gps-sensor-name-2>"],
+        "orientation" : ["<your-imu-sensor-name-1>"],
+        "compass_heading" : ["<your-gps-sensor-name-1>"],
+        "angular_velocity":["<your-angular-velocity-sensor-name-2>"],
+        "linear_velocity": ["<your-gps-sensor-name-1>"],
+        "linear_acceleration": ["<your-adxl345-sensor-name-1>"]
+      },
+    "depends_on": [] 
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+// { "components": [ { ... },
+{
+    "name": "<your-sensor-name>",
+    "type": "movement_sensor",
+    "model": "merged",
+    "attributes": {
+        "position": ["gps1"],
+        "orientation" : ["vectornav"],
+        "compass_heading" : ["gps1"],
+        "angular_velocity":[""],
+        "linear_velocity": ["gps1"],
+        "linear_acceleration": ["adxl345"]
+    }, 
+      "depends_on":[]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Attributes
+
+Configure an array of the `name` of each movement sensor you want to add to your robot as a merged resource in the attributes of the `merged` movement sensor model:
+
+- The name of each attribute represents the `Property` that that particular movement sensor supports, or the type of reading or measurement that it takes.
+- Retrieve this information for your movement sensor by calling [`GetProperties()`](/components/movement-sensor/#getproperties) on the sensor.
+- Use this information to determine which attribute to put its `name` inside the array of.
+
+Name | Type | Inclusion | Description
+---- | ---- | --------- | -----------
+`position` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads position. |
+`orientation` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads orientation. |
+`compass_heading` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads compass heading position. |
+`linear_velocity` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads linear velocity. |
+`angular_velocity` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads angular velocity. |
+`linear_acceleration` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads linear acceleration |
+
+Note that `Readings` are only taken and merged from the first sensor in the respective attribute's array that does not produce an error at runtime.
```

Comment:
```suggestion
Note that readings are only available from the first sensor in the respective attribute's array that does not produce an error at runtime.
```

Reply:
Ahh I just saw your main review comment notes on here!! I think you answered this already so if you are re-reviewing and see this, sorry, premature lol

Reply:
Ok-- I rephrased hopefully more accurate, comment still stands as far as keeping note in I guess!